src/App.tsx

```diff
@@ -1,21 +1,44 @@
 import "./App.css";
-import React, { useState } from "react";
-import { taskList } from "./taskList";
-
+import React, { useState, useEffect } from "react";
 import icon from "./icons8-delete.svg";
 
-
-//
 function TaskShow() {
-// Create a list with additional field to mark deleted task
-  const [tasks, setTasks] = useState(
-    taskList.map((task) => ({
-      id: task.id,
-      name: task.name,
-      isDeleted: false,
-    }))
-  );
-  const [newTaskName, setNewTaskName] = useState("");
+
+  interface Item {
+    id: number;
+    name: string;
+  }
+  interface ExItem {
```

Comment:
this name is not clear, what does `Ex` mean? Maybe `ItemWithDeleteFlag` is a better name